docs/source/reference/migration/from-router-v1.mdx

```diff
@@ -188,6 +204,14 @@ telemetry:
 
 - `apollo_router_session_count_total` has been removed and replaced by `http.server.active_requests`.
 
+- `apollo_require_authentication_failure_count` has been removed. Use the `http.server.request.duration` metric's `http.response.status_code` attribute. Requests with authentication failures have HTTP status code 401.
+
+- `apollo_authentication_failure_count` has been removed. Use the `apollo.router.operations.authentication.jwt` metric's `authentication.jwt.failed` attribute.
+
+- `apollo_authentication_success_count` has been removed. Use the `apollo.router.operations.authentication.jwt` metric instead. If the `authentication.jwt.failed` attribute is _absent_ or `false`, the authentication succeeded.
+
+- `apollo_router_deduplicated_subscriptions_total` has been removed. Use the `apollo.router.operations.subscriptions` metric's `subscriptions.deduplicated` attribute
```

Comment:
I am not certain how docs are going to be built for 1.x vs 2.x. Is dev explicitly tied to 2.x docs now? Should I go ahead and remove/tweak the actual docs referencing these metrics?

Reply:
`dev` is the 2.x docs now. 1.x docs will come from `1.x`. So, yes you can update docs.

Reply:
excellent! done in be696bcb5da1235069a8d910b2a9b13f504375a1